rosdep/python.yaml

```diff
@@ -2485,6 +2485,10 @@ python-networkx:
   gentoo: [dev-python/networkx]
   ubuntu:
     '*': [python-networkx]
+python3-nmcli:
```

Comment:
Please name this `python3-nmcli-pip`: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1